pkg/providers/instance/instance.go

```diff
@@ -284,11 +301,17 @@ func newVMObject(
 	nodeIdentities []string,
 	nodeClass *v1alpha2.AKSNodeClass,
 	launchTemplate *launchtemplate.Template,
-	instanceType *corecloudprovider.InstanceType) armcompute.VirtualMachine {
+	instanceType *corecloudprovider.InstanceType,
+	provisionMode string) armcompute.VirtualMachine {
 	// Build the image reference from template
 	imageReference := armcompute.ImageReference{
 		CommunityGalleryImageID: &launchTemplate.ImageID,
 	}
+
+	if launchTemplate.IsWindows {
+		return armcompute.VirtualMachine{} // TODO(Windows)
+	}
```

Comment:
why add this wiring here now if you arent going to support it anywhere? Assuming users will enable windows through the AKS NodeClass and a new ImageFamily api value. 

Wouldnt using v1alpha2.AKSNodeClass.Spec.ImageFamily make more sense rather than putting this in the template?

Reply:
It was already half-done so I wouldn't want to remove it if we might add them again. Doubt it is too distracting in practice. But could still be moved elsewhere by then.